### ISSUE TITLE
Update the projects dynamic parts

### ISSUE THREAD

**Issue by rwaDept** [Kentico/kontent-sample-app-vue#60]
Today I setup this sample project for the first time. I'm new to this product and would use this sample project as a way to find my way around Kentiko Kontent / SaaS. I used the documentation page as a guide (https://docs.kontent.ai/tutorials/develop-apps/get-started/run-sample-app?tech=javascript#a-making-changes-to-your-project). 

After the successful first run I of course go and try to change an item in the CMS in order to see that change being reflected on my sample site. Unfortunately I was disappointed not to see this change happen. After a talk with your excellent in app chat and digging around a little myself I come to the realisation that not every part of this sample project is being loaded dynamically from the CMS. 

I understand the need to show that you can combine the dynamic pages with static ones. However since it's main purpose is showing off the connection with Kentiko Kontent I'd at least expect the home page (the first page you see) to be dynamic. Personally I'd create a separate section of the site to show off the static parts. But barring that, it would have been nice to know at least that not all parts are actually dynamic, and which parts are. 

So in short; at least add some documentation somewhere (where it can't be missed) stating that not all content in the sample project is being loaded dynamically, where to find that static content in the code and additionally maybe state which content parts are dynamic and which are not. 

Lastly, maybe move the entire static part to its own section. Though as long as it's clear where the content comes from this is less of an issue. I also understand the need for a page with a mixture of dynamic / non-dynamic content.

**Comment by Simply007** [comment]
Hello @rwaDept,

the first step would be to point people how to distinguish the data loaded from Kontent and ones loaded from JSONs.

Do you think it would be sufficient to make a note to the readme that?
*  All data loaded from Kentico Kontent are using [`Stores`](https://github.com/Kentico/kontent-sample-app-vue/tree/master/src/Stores) as for examples "Latest Articles" section ([component here](https://github.com/Kentico/kontent-sample-app-vue/blob/master/src/components/LatestArticles.vue#L111)) on Home Page 
* The data provided from [`Localization`](https://github.com/Kentico/kontent-sample-app-vue/tree/master/src/Localization) are using `vue-i18n` plugin, so every call in components using `$t('KEYWORD')` in components is loading data from these JSONs as for example "Banner" section" ([component here](https://github.com/Kentico/kontent-sample-app-vue/blob/master/src/components/Banner.vue#L6))

**Comment by rwaDept** [comment]
Hello @Simply007,

Yes that's a good first step, adding that to the readme here on GitHub. Though, as long as the dynamic and JSON content is mixed ideally I'd say it's a good idea to also show that on the sample site itself (maybe a notification inside the content itself?). Not everyone will open the code before checking the connection and thus might not see the localisation parts.
